statsmodels/graphics/functional.py

```diff
@@ -392,7 +410,8 @@ def _band_quantiles(band):
 
         pool = Pool()
         data = zip(range(dim), itertools.repeat((band, pca,
-                                                 bounds, ks_gaussian)))
+                                                 bounds, ks_gaussian,
+                                                 seed, use_brute)))
```

Comment:
Doesn't this use the same seed in each call/repeat?
causing perfect correlation across `dim`

(I never tried to understand the details here.)

Reply:
seed is just used for differential evolution, so maybe/likely it doesn't matter.

Reply:
No, it ultimately calls

https://github.com/statsmodels/statsmodels/blob/c5bf4a6338bea8cdc37b931101cbe133f30543d9/statsmodels/graphics/functional.py#L94

which takes idx as a parameter (which comes from range).  

I'm not trying to fix anything other than to allow it to be deterministic.

Reply:
But AFAICS it changes the differential evolution behavior.
Before it was using the global random state which continuous the random chain with each call.
Now it uses the same seed in each call to differential_evolution.

I don't think anymore that it matters in this case, but in general using identical seed would be incorrect, e.g. in bootstrap or parallel randomized optimization for the same function. The latter is the case that I thought about initially.